package.json

```diff
@@ -70,47 +70,47 @@
     "visual-snapshot": "cd packages/core && yarn percy storybook storybook-static"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.9",
-    "@babel/eslint-parser": "^7.23.9",
-    "@babel/preset-react": "^7.17.12",
-    "@commitlint/cli": "^18.6.0",
-    "@commitlint/config-conventional": "^18.6.0",
+    "@babel/core": "^7.26.0",
+    "@babel/eslint-parser": "^7.26.5",
+    "@babel/preset-react": "^7.26.3",
+    "@commitlint/cli": "^19.6.1",
+    "@commitlint/config-conventional": "^19.6.0",
     "@playwright/test": "^1.49.1",
     "@testing-library/dom": "^8.11.4",
     "@testing-library/react": "^14.0.0",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
     "@types/carbon__layout": "^0.0.3",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
-    "accessibility-checker": "^3.1.78",
-    "cheerio": "^1.0.0-rc.12",
-    "commander": "^12.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.20.0",
+    "@typescript-eslint/parser": "^8.20.0",
+    "accessibility-checker": "^3.1.79",
+    "cheerio": "^1.0.0",
+    "commander": "^13.0.0",
     "copyfiles": "^2.4.1",
-    "cspell": "^8.3.2",
-    "eslint": "^8.56.0",
+    "cspell": "^8.17.2",
+    "eslint": "^9.18.0",
```

Comment:
We're going to need to downgrade back to `^8.56.0` until we investigate https://github.com/carbon-design-system/carbon/issues/18305 more